tests/resources.test

```diff
@@ -28,8 +31,12 @@ for d in /dev/shm /var/run $SOCKETDIR; do
 	if [ "$(printf '%s\n' "${leftovers}" | wc -l)" -eq $(($SOCKS_PER_PROCESS * 2)) ]; then
 		echo
 		echo "There were some empty leftovers (expected), removing them"
-		echo "${leftovers}" | tee /dev/stderr | xargs rm
+		echo "${leftovers}" | tee /dev/stderr
```

Comment:
It seems odd to output just the names to both stdout and stderr, I would expect all the echos to go to one place. echo blah >&2 goes to just stderr